compiler/luci-interpreter/src/Interpreter.cpp

```diff
@@ -125,6 +125,21 @@ void Interpreter::readOutputTensor(const luci::CircleOutput *output_node, void *
     tensor->readData(data, data_size);
 }
 
+size_t Interpreter::getOutputTensorSize(const luci::CircleOutput *output_node)
+{
+  Tensor *tensor = _runtime_module->getOutputTensors()[output_node->index()];
+  if (tensor == nullptr)
+  {
+    const std::string &name = output_node->name();
+    throw std::runtime_error("Cannot find tensor size for output node named \"" + name + "\".");
+  }
+
+  size_t tensor_size = luci::size(output_node->dtype());
+  for (int i = 0; i < tensor->shape().num_dims(); i++)
+    tensor_size *= tensor->shape().dim(i);
```

Comment:
There is an API for this purpose.

```suggestion
  tensor_size *= tensor->shape().num_elements();
```

Reply:
Thank you!